[BUG] Return value is unexpected tuple  if  create collection timeout

```
        status = connect.create_collection(param, timeout=timeout)
>       if not status.OK():
E       AttributeError: 'tuple' object has no attribute 'OK'

conftest.py:104: AttributeError
```

**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Environment details**

- Hardware/Softward conditions (OS, CPU, GPU, Memory)
- Method of installation (Docker, or from source)
- Milvus version (v0.3.1, or v0.4.0)
- Milvus configuration (Settings you made in `server_config.yaml`)

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,8 @@
 # pymilvus 0.2.11(TBD)
 
+## Bug
+- \#190 - Fix rpc return error when error occur
+
 
 # pymilvus 0.2.10(2020-04-15)
 

--- a/examples/example.py
+++ b/examples/example.py
@@ -24,12 +24,11 @@ _INDEX_FILE_SIZE = 32  # max file size of stored index
 
 
 def main():
+    # Specify server addr when create milvus client instance
     milvus = Milvus(_HOST, _PORT)
 
     # Check if server is accessible
-    if not milvus.ping():
-        print("Server is unreachable")
-        sys.exit(0)
+    milvus.connect()
 
     # Create collection demo_collection if it dosen't exist.
     collection_name = 'example_collection_'

--- a/examples/example_binary.py
+++ b/examples/example_binary.py
@@ -35,8 +35,8 @@ def gen_binary_vectors(dim, num):
 
 
 def main(collection_name, metric_type):
-    client = Milvus()
-    client.connect(host="localhost", port="19530")
+    client = Milvus(host="localhost", port="19530")
+    client.connect()
 
     _, ok = client.has_collection(collection_name)
 

--- a/examples/example_partition.py
+++ b/examples/example_partition.py
@@ -22,6 +22,8 @@ def main():
     # You may need to change _HOST and _PORT accordingly
     param = {'host': _HOST, 'port': _PORT}
 
+    # You can create a instance specified server addr and
+    # invoke rpc method directly
     client = Milvus(**param)
     # Create collection demo_collection if it dosen't exist.
     collection_name = 'demo_partition_collection'

--- a/examples/multiprocess_example.py
+++ b/examples/multiprocess_example.py
@@ -29,8 +29,8 @@ def _generate_vectors(_dim, _num):
 
 def prepare_collection(_collection_name):
     def _create_collection(_collection_param):
-        milvus = Milvus()
-        milvus.connect(**server_config)
+        milvus = Milvus(**server_config)
+        milvus.connect()
         status, ok = milvus.has_collection(_collection_name)
         if ok:
             print("Table {} found, now going to delete it".format(_collection_name))
@@ -62,8 +62,8 @@ def multi_insert(_collection_name):
     process_list = []
 
     def _add():
-        milvus = Milvus()
-        status = milvus.connect(**server_config)
+        milvus = Milvus(**server_config)
+        status = milvus.connect()
 
         vectors = _generate_vectors(128, 10000)
         print('\n\tPID: {}, insert {} vectors'.format(os.getpid(), 10000))

--- a/milvus/client/grpc_handler.py
+++ b/milvus/client/grpc_handler.py
@@ -36,11 +36,11 @@ def error_handler(*rargs):
             except grpc.FutureTimeoutError as e:
                 LOGGER.error("{}\n{}".format(func.__name__, e))
                 status = Status(Status.UNEXPECTED_ERROR, message='Request timeout')
-                return tuple([status]) + rargs
+                return status if not rargs else tuple([status]) + rargs
             except grpc.RpcError as e:
                 LOGGER.error("{}\n{}".format(func.__name__, e))
                 status = Status(e.code(), message='Error occurred. {}'.format(e.details()))
-                return tuple([status]) + rargs
+                return status if not rargs else tuple([status]) + rargs
 
         return handler
 

--- a/tests/test_collection.py
+++ b/tests/test_collection.py
@@ -1,7 +1,10 @@
-import pytest
 import sys
-
 sys.path.append(".")
+from unittest import mock
+
+import grpc
+import pytest
+from grpc._channel import _UnaryUnaryMultiCallable as uum
 
 from milvus import Milvus, MetricType, IndexType, ParamError
 
@@ -97,6 +100,18 @@ class TestCreateCollection:
         }
         with pytest.raises(ParamError):
             gcon.create_collection(collection_param)
+
+    # @mock.patch("grpc._channel._UnaryUnaryMultiCallable")
+    # def test_create_collection_exception(self, mock_future, gcon):
+    #     mock_future.future = mock.Mock(side_effect=grpc.FutureTimeoutError())
+    #     collection_param = {
+    #         "collection_name": "test_create_collection_exception",
+    #         "dimension": 128,
+    #         "metric_type": MetricType.L2,
+    #         "index_file_size": 10
+    #     }
+    #     with pytest.raises(Exception):
+    #         gcon.create_collection(collection_param)
 
 
 class TestHasCollection:

